### PR TITLE
Multiple tables

### PR DESCRIPTION
**Pull request by Danieldotcomcoder**
Use primary key & foreign key mechanisms for joining tables.
Query multiple tables.
Prepare complex queries that answer analytical questions.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -109,4 +109,51 @@ SELECT
...
\ No newline at end of file
+GROUP BY species
+
+--------------------------------------------------------------------------------------------------------------------------
+
+-- What animals belong to Melody Pond?
+
+SELECT * FROM animals
+JOIN owners ON animals.owner_id = owners.id
+WHERE animals.owner_id = (SELECT id from owners WHERE full_name='Jennifer Orwell');
```
Comment (AshakaE) [inline queries.sql]: - [ ] Please correct the full name here, I believe it's 'Melody Pond'

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @Danieldotcomcoder  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Danieldotcomcoder) [reply queries.sql]: Thank you , I did Fix the name

**Review by AshakaE — APPROVED** [review]
### Hi @Danieldotcomcoder   🙋‍♀️, 

# STATUS APPROVED ✔

 Thanks for making the change requested 👍 
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**